pkg/workflows/validate.go

```diff
@@ -0,0 +1,91 @@
+package workflows
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/config"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/kubeconfig"
+	"github.com/aws/eks-anywhere/pkg/types"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/validations/cmdvalidations"
+	"github.com/aws/eks-anywhere/pkg/validations/createvalidations"
+)
+
+type Validate struct {
```

Comment:
I'm not sure I understand what this is supposed to be
Usually it's easier to understand what an entity is when named with a noun